spec/pom.xml

```diff
@@ -61,6 +60,7 @@
                             <goal>process-asciidoc</goal>
                         </goals>
                         <configuration>
+                            <outputFile>${project.build.directory}/generated-docs/${project.build.finalName}.pdf</outputFile>
```

Comment:
why do you define another `<outputFile>`?

Reply:
I suggest to define a property 
`<config-spec-pdf>${project.build.directory}/generated-docs/${project.build.finalName}.pdf</config-spec-pdf>` so that the following places can refer to that property

Reply:
One if for the pdf and the other one is for html.

Reply:
Ah nvm I think when I did the change I didn't got yours so when it rebase it merged both.